starknet_py/tests/e2e/account/account_client_test.py

```diff
@@ -128,3 +128,50 @@ async def test_estimate_fee_called(run_devnet):
         ).estimate_fee()
 
         mocked_estimate_fee.assert_called()
+
+
+@pytest.mark.asyncio
+async def test_estimated_fee_grater_than_zero(run_devnet):
+    acc_client = await DevnetClientFactory(run_devnet).make_devnet_client()
+
+    deployment_result = await Contract.deploy(
+        client=acc_client, compilation_source=erc20_mock_source_code
+    )
+    deployment_result = await deployment_result.wait_for_acceptance()
+    erc20_contract = deployment_result.deployed_contract
+
+    estimated_fee = (
+        await erc20_contract.functions["balanceOf"]
+        .prepare("1234", max_fee=0)
+        .estimate_fee()
+    )
+
+    assert estimated_fee > 0
+
+
+@pytest.mark.asyncio
+async def test_fee_higher_for_account_client(run_devnet):
+    acc_client = await DevnetClientFactory(run_devnet).make_devnet_client()
+    client = await DevnetClientFactory(run_devnet).make_devnet_client_without_account()
+
+    deployment_result = await Contract.deploy(
+        client=acc_client, compilation_source=erc20_mock_source_code
+    )
+    deployment_result = await deployment_result.wait_for_acceptance()
+
+    contract_acc = deployment_result.deployed_contract
+    contract_client = await Contract.from_address(contract_acc.address, client)
+
+    estimated_fee_acc = (
```

Comment:
I know you used `_acc` and `_client` below to be consistent with the rest of variables in this test, but perhaps naming them `estimated_fee_signed` and `estimated_fee` would make it more explicit about what this is actually testing.

Consider if this test failed, and you'd get something like `assertion failed: estimated_fee_client not less than estimated_fee_acc` - that doesn't tell much.